guacamole-rdp-vnc-gateway-existing-vnet/azuredeploy.json

```diff
@@ -0,0 +1,229 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
```

Comment:
Since this is the location of existing VNET, then it's best to call this parameter "existingVNETLocation" to set the right expectation